sunpy/map/tests/conftest.py

```diff
@@ -102,13 +102,14 @@ def generic_map():
 
 
 def make_simple_map():
```

Comment:
```suggestion
@pytest.fixture
def simple_map():
```
(and remove the `simple_map = ...` below)

I don't understand why this isn't made a fixture via decorator like every other fixture we define.  You'll also need to modify the following lines, but those tests should really be using the fixture anyway:
https://github.com/sunpy/sunpy/blob/2b416fdacb755aaf9cf94b960a9995b2997758a4/sunpy/map/tests/test_mapbase.py#L39
https://github.com/sunpy/sunpy/blob/2b416fdacb755aaf9cf94b960a9995b2997758a4/sunpy/map/tests/test_mapbase.py#L1028
https://github.com/sunpy/sunpy/blob/2b416fdacb755aaf9cf94b960a9995b2997758a4/sunpy/map/tests/test_mapbase.py#L1049